.github/workflows/check-submodule-pointers.yml

```diff
@@ -0,0 +1,32 @@
+name: Check submodule pointers
+
+on:
+  push:
+    branches: [ main ]
+  schedule:
+    - cron: '42 12 * * *'
+
+jobs:
+  check-submodules:
+    runs-on: ubuntu-latest
+    if: github.repository == 'github/vscode-codeql-starter'
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Compare submodule pointers to lgtm.com branch
+        env: 
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+            CODEQL_ACTUAL_SHA="$(git rev-parse @:./ql)"
```

Comment:
Had to look this up. Never seen this syntax before. Good to learn something new.